class-gf-connectwise-2020v1.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+require_once WP_PLUGIN_DIR . '/connectwise-forms-integration/class-gf-connectwise.php';
+GFForms::include_feed_addon_framework();
+
+class GFConnectWise2020v1 extends GFConnectWise {
+	private static $_instance = null;
+
+	public static function get_instance() {
+		if ( null == self::$_instance ) {
+			self::$_instance = new self;
+		}
+
+		return self::$_instance;
+	}
+
+	public function get_team_members() {
+		$this->log_debug( __METHOD__ . '(): start getting team members from ConnectWise' );
```

Comment:
Just wonder, where does this log goes to? In the console?

Reply:
<img width="254" alt="Screen Shot 2563-02-19 at 15 15 20" src="https://user-images.githubusercontent.com/19608954/74814896-ceb63180-532a-11ea-92ff-f87e13ebc058.png">

Reply:
it show in the log file
<img width="809" alt="Screen Shot 2563-02-19 at 15 15 26" src="https://user-images.githubusercontent.com/19608954/74815449-d2968380-532b-11ea-9997-5ce0c76a1faf.png">